src/main/java/banking/account/controller/BankController.java

```diff
@@ -0,0 +1,39 @@
+package banking.account.controller;
+
+import banking.account.domain.BasicAccount;
+import banking.account.dto.AccountDTO;
+import banking.account.service.BankService;
+import banking.account.service.Service;
+
+import java.math.BigDecimal;
+
+
+public class BankController {
+    // 싱글톤 패턴
+    public static final BankController bankController = new BankController();
```

Comment:
싱글톤 패턴을 사용할 때 외부에서 직접적인 필드 접근은 허용되어서는 안됩니다
```suggestion
    private static final BankController bankController = new BankController();
```